primitives/rpc/src/lib.rs

```diff
@@ -215,6 +227,19 @@ sp_api::decl_runtime_apis! {
 		#[changed_in(2)]
 		fn convert_transaction(transaction: ethereum::TransactionV0) -> <Block as BlockT>::Extrinsic;
 	}
+
+	#[api_version(2)]
```

Comment:
so why use `api_version(2)` here, it's a new runtime API.
about compatibility for moonbeam?

Reply:
Nice catch, this is a copy-paste from the moonbeam repo. Considering that the txpool only related to the txpool pending transaction, I think there is no compatibility issue for the moonbeam if we remove the version here.  @tgmichel  Please confirm this.

Reply:
versioning is not needed here you can safely remove it

Reply:
@tgmichel I think we can utilize the `extrinsic_filter` in the `EthereumRuntimeRPCApi`,  `TxPoolRuntimeApi` is no longer needed.

Reply:
Ok. Let me fix it tomorrow.